src/eth/executor.rs

```diff
@@ -38,9 +50,15 @@ impl EthExecutor {
         }
     }
 
-    /// Execute a transaction, mutate the state and return function output.
+    /// Executes Ethereum transactions and facilitates block creation.
```

Comment:
It would be useful to have similar documentation for the `call` function because it is used in `eth_call` and `eth_estimateGas` operations.